From: Jinoh Kang <jinoh.kang.kr@gmail.com>
Subject: [PATCH] ntdll: Map ETXTBSY to STATUS_SHARING_VIOLATION.
Message-Id: <88fe4a89-c262-bf66-5998-a9a3d0163e60@gmail.com>
Date: Tue, 8 Feb 2022 20:57:44 +0900

This complements commit c75edd04c2c (server: Map ETXTBSY to
STATUS_SHARING_VIOLATION., 2007-09-11).

Signed-off-by: Jinoh Kang <jinoh.kang.kr@gmail.com>
---
 dlls/ntdll/unix/file.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/dlls/ntdll/unix/file.c b/dlls/ntdll/unix/file.c
index 11111111111..11111111111 100644
--- a/dlls/ntdll/unix/file.c
+++ b/dlls/ntdll/unix/file.c
@@ -540,6 +540,7 @@ NTSTATUS errno_to_status( int err )
     TRACE( "errno = %d\n", err );
     switch (err)
     {
+    case ETXTBSY:
     case EAGAIN:    return STATUS_SHARING_VIOLATION;
     case EBADF:     return STATUS_INVALID_HANDLE;
     case EBUSY:     return STATUS_DEVICE_BUSY;

-- 
0.0.0

